rust_src/admin/remacs-helpers.el

```diff
@@ -0,0 +1,52 @@
+;;; remacs-helpers.el -- tools for Remacs development
+;;; Commentary:
+
+;; This is a collection of tools to help developers working with Remacs source.
+
+(require 'dash)
+(require 's)
+
+;;; Code:
+
+(defun remacs-helpers/ignored-type-part-p (input)
+  "Predicate to indicate if INPUT is part of a C type ignored in Rust."
+  (string= input "struct"))
+
+(defun remacs-helpers/make-rust-args-from-C-worker (input)
+  "Transform C function arguments INPUT into Rust style arguments.
+
+For example, convert this:
+int *foo, struct Bar b
+
+into this:
+foo: *mut int, b: Bar"
+  (->> input
+       (s-split ",")
+       (-map #'s-trim)
+       (--map (-remove 'remacs-helpers/ignored-type-part-p (s-split " " it)))
+       (-map (-lambda ((type name))
+               (if (s-starts-with-p "*" name)
+                   (format "%s: *mut %s" (s-chop-prefix "*" name) type)
+                 (format "%s: %s" name type))))
+       (s-join ", ")))
+
```

Comment:
This doesn't work if the argument contains the `register` keyword.

Reply:
Yup. That is why I have the predicate to drop parts....